scripts/devaccess/devaccess_wrap.py

```diff
@@ -289,6 +289,19 @@ def oc_get_pods(occmd):
 
         return results
 
+    @staticmethod
+    def oc_get_routes(occmd):
+        ''' provide 'oc get routes' '''
+
+        n_cmd = occmd.normalized_cmd()
+        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
+        # -o yaml and -o json output of routes can expose
```

Comment:
Agreed -- we should look to redact this, or perhaps file a feature request upstream for "oc get" to filter sensitive information either by default or when asked.

For the sake of tiered access, I'll +1 anyway, with the expectation that anyone granted access to this command is informed of the privacy implications.